### PR TITLE
add paragraph about remote disk impact

### PR DESCRIPTION
**Pull request by rgbj**

### CODE REVIEW

File: pages/platform/databases/cassandra_01_capabilities/guide.de-de.md [hunk 1]
```diff
@@ -105,6 +105,13 @@ Here are the node types you can choose from:
 
 Right now, all nodes of a given cluster should be of the same type and live in the same region.
 
+#### Disk type
+
+The type of storage available may vary according to the region your cluster lives in. Thus, your cluster may be backed by e.g. *High Speed* or *High Speed Gen2* block storage.
+
+Also, the performance caracteristics of the various storage offerings may vary depending on e.g. the storage size your cluster uses: *High Speed* may offer better iops than *High Speed Gen2* for some disk sizes.
+
+Public Cloud Databases will select the most efficient disk type for your cluster depending on your cluster parameters.
```
Comment (BellionBastien) [inline pages/platform/databases/cassandra_01_capabilities/guide.de-de.md:114]: Add this link for customer to see in which region which storage technology is available: https://www.ovhcloud.com/en/public-cloud/regions-availability/

File: pages/platform/databases/cassandra_01_capabilities/guide.de-de.md [hunk 2]
```diff
@@ -105,6 +105,13 @@ Here are the node types you can choose from:
 
 Right now, all nodes of a given cluster should be of the same type and live in the same region.
 
+#### Disk type
+
+The type of storage available may vary according to the region your cluster lives in. Thus, your cluster may be backed by e.g. *High Speed* or *High Speed Gen2* block storage.
```
Comment (BellionBastien) [inline pages/platform/databases/cassandra_01_capabilities/guide.de-de.md]: Add this link for customer to see performance differences between technology: https://www.ovhcloud.com/fr/public-cloud/block-storage/

File: pages/platform/databases/cassandra_01_capabilities/guide.de-de.md [hunk 3]
```diff
@@ -105,6 +105,13 @@ Here are the node types you can choose from:
 
 Right now, all nodes of a given cluster should be of the same type and live in the same region.
 
+#### Disk type
+
+The type of storage available may vary according to the region your cluster lives in: see https://www.ovhcloud.com/en/public-cloud/regions-availability/ for more information about block storage type availability depending on region. Thus, your cluster may be backed by e.g. *High Speed* or *High Speed Gen2* block storage.
```
Comment (OVH-Team-Guides) [inline pages/platform/databases/cassandra_01_capabilities/guide.de-de.md]:
```suggestion
The type of storage available may vary according to the region your cluster lives in: see <https://www.ovhcloud.com/en/public-cloud/regions-availability/> for more information about block storage type availability depending on region. Thus, your cluster may be backed by e.g. *High Speed* or *High Speed Gen2* block storage.
```

File: pages/platform/databases/cassandra_01_capabilities/guide.de-de.md [hunk 4]
```diff
@@ -105,6 +105,13 @@ Here are the node types you can choose from:
...
+
+Also, the performance caracteristics of the various storage offerings may vary depending on e.g. the storage size your cluster uses: *High Speed* may offer better iops than *High Speed Gen2* for some disk sizes. See https://www.ovhcloud.com/fr/public-cloud/block-storage/ for more information about those performance caracteristics.
```
Comment (OVH-Team-Guides) [inline pages/platform/databases/cassandra_01_capabilities/guide.de-de.md]:
```suggestion
Also, the performance caracteristics of the various storage offerings may vary depending on e.g. the storage size your cluster uses: *High Speed* may offer better iops than *High Speed Gen2* for some disk sizes. See <https://www.ovhcloud.com/fr/public-cloud/block-storage/> for more information about those performance caracteristics.
```

**Review by OVH-Team-Guides — CHANGES_REQUESTED** [review]
Hello @rgbj , could you please make all the URLs clickable in all the updated guides ? Thanks

File: pages/platform/databases/cassandra_01_capabilities/guide.de-de.md (hunk 3 above)
Comment (rgbj) [reply pages/platform/databases/cassandra_01_capabilities/guide.de-de.md]: done

File: pages/platform/databases/cassandra_01_capabilities/guide.de-de.md (hunk 4 above)
Comment (rgbj) [reply pages/platform/databases/cassandra_01_capabilities/guide.de-de.md]: done

**Review by OVH-Team-Guides — APPROVED** [review]

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @rgbj

**Pull request merged**